firmware/controllers/map_averaging.cpp

```diff
@@ -240,8 +271,10 @@ float getMap(void) {
 #if EFI_ANALOG_SENSORS || defined(__DOXYGEN__)
 	if (!isValidRpm(engine->rpmCalculator.rpmValue))
 		return validateMap(getRawMap()); // maybe return NaN in case of stopped engine?
-	return validateMap(getMapByVoltage(v_averagedMapValue));
-#else
+	if (mapMinBufferLength > 1)
```

Comment:
I believe this if-statement is not needed, we can simplify the code by throwing both if-statements away...